src/constants.zig

```diff
@@ -277,19 +277,41 @@ pub const grid_repair_request_max = config.process.grid_repair_request_max;
 /// The number of grid reads allocated to handle incoming command=request_blocks messages.
 pub const grid_repair_reads_max = config.process.grid_repair_reads_max;
 
-/// The number of grid writes allocated to handle incoming command=block messages.
-pub const grid_repair_writes_max = config.process.grid_repair_writes_max;
+/// Immediately after state sync we want access to all of the grid's write bandwidth to rapidly sync
+/// table blocks.
+pub const grid_repair_writes_max = grid_iops_write_max;
 
 /// The default sizing of the grid cache. It's expected for operators to override this on the CLI.
 pub const grid_cache_size_default = config.process.grid_cache_size_default;
 
+/// The maximum capacity (in blocks) of the GridRepairQueue.
```

Comment:
:thinking: Hmm, good point, that is confusing. I think I'll go with `GridMissingBlocks` though -- I want "Grid" in there to make the relationship with the `Grid` obvious.

(I was considering `GridRequestQueue` as an alternative, but we have other "request queues" elsewhere, and the thing being queued is missing blocks, not requests, so that more makes sense.)

Reply:
Another idea: `GridGaps`

Reply:
:thinking: "Grid gaps" makes me think of the unallocated parts of the grid, though.